autopark/VehicleType.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace autopark
+{
+    class VehicleType
+    {
+        string typeName;
```

Comment:
You don't have access to this field. Create Property for that field or replace in with Auto Property like that.
```suggestion
        public string TypeName { get; set; }
```

Reply:
![image](https://user-images.githubusercontent.com/79978383/115709693-30808a00-a37a-11eb-9c4a-d34de6fe82b9.png)
Done